test/e2e/tls_test.go

```diff
@@ -115,14 +118,14 @@ func (data *TestData) checkTLS(t *testing.T, podName string, containerName strin
 	}
 
 	// 2. Set TLSMaxVersion to TLS1.2, then TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256 should be used
-	stdouts = data.opensslConnect(t, podName, containerName, true, apiserver)
+	stdouts = data.opensslConnect(t, podName, containerName, true, apiserver, podIPv4, podIPv6)
 	for _, stdout := range stdouts {
 		assert.True(t, strings.Contains(stdout, fmt.Sprintf("New, TLSv1.2, Cipher is %s", cipherSuiteStr)),
 			"Cipher Suite used by %s apiserver should be the TLS1.2 one '%s', output: %s", apiserverStr, cipherSuiteStr, stdout)
 	}
 }
 
-func (data *TestData) opensslConnect(t *testing.T, pod string, container string, tls12 bool, port int) []string {
+func (data *TestData) opensslConnect(t *testing.T, pod string, container string, tls12 bool, port int, podIPv4, podIPv6 string) []string {
```

Comment:
You can check ipv4/ipv6 string to append necessary case to a variable of test struct "opensslConnectCommands".